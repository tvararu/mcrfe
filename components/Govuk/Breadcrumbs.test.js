import renderer from "react-test-renderer";
import Breadcrumbs from "./Breadcrumbs";
jest.mock("next/link");

describe("Breadcrumbs", () => {
  it("renders correctly", () => {
    const crumbs = [
      { href: "/", text: "Organisations" },
      { text: "An organisation name" }
    ];
    const component = renderer.create(<Breadcrumbs crumbs={crumbs} />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
