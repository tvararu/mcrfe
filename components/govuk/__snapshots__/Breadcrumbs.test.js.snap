// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumbs renders correctly 1`] = `
<nav
  className="govuk-breadcrumbs"
>
  <ol
    className="govuk-breadcrumbs__list"
  >
    <li
      className="govuk-breadcrumbs__list-item"
    >
      <a
        className="govuk-breadcrumbs__link"
      >
        Organisations
      </a>
    </li>
    <li
      aria-current="page"
      className="govuk-breadcrumbs__list-item"
    >
      An organisation name
    </li>
  </ol>
</nav>
`;
